docs/modules/angular/pages/cookbooks/vanilla-to-onecx/migrate-to-onecx.adoc

```diff
@@ -0,0 +1,279 @@
+= Migrate vanilla Angular apps to OneCX
+Bastian Jakobi bastian.jakobi@capgemini.com
+:source-highlighter: highlight.js
+
+The goal of this guide is to provide step by step instructions on how to migrate a vanilla Angular app (e.g. generated with Angular CLI) to a OneCX compatible microfrontend application. 
+
+== General
+=== Prerequisites
+This guide assumes that an Angular project is already set up and that a OneCX platform instance is up and running. At the time of writing, some required dependencies are only compatible with Angular version 18. Therefore the application must be updated to Angular 18 before migrating. Additionally, the Angular application should be based on feature modules, which are lazy loaded before routing.
+
+Note that this guide does not cover general Angular project setup or the steps to configure OneCX.
+
+== Migration steps
+The goal of this migration is to transform a "vanilla" Angular app into a OneCX compatible app that exposes a microfrontend which can be loaded and displayed by the OneCX platform. To achieve this goal, a few migration steps have to be completed. All steps have been tested in a fresh app generated with Angular CLI but should also work in other Angular apps (e.g. apps generated with NX).
+
+=== Ensure that your Angular app runs in standalone mode
```

Comment:
Find a better term for standalone because it can be confused with the standalone mode in the other guide